CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+- Provide as much context as possible about the issue you're facing.
+
+We will address your issue as soon as possible.
+
+### Project Communications
+
+Most project communications occur on our [Discord](https://discord.gg/fsEW23wFYs) server. Discussions related to the benchmarking tool typically happen in the [benchmarking-tool](https://discord.gg/Kv6uucUq) channel.
+
+Discussions about specific codebase work take place on GitHub, in [issues](https://github.com/stratum-mining/benchmarking-tool/issues) and [pull requests](https://github.com/stratum-mining/benchmarking-tool/pulls).
+
+Our development calls are scheduled every Tuesday at 18:00 CET. You can see them in the Discord sidebar under Events and subscribe to be notified.
+
+## I Want To Contribute
+
+> When contributing to this project, you must agree that you have authored 100% of the content, that you have the necessary rights to the content, and that the content you contribute may be provided under the project license.
+
+### Contribution Workflow
+
+The SRI project follows an open contributor model, where anyone is welcome to contribute through reviews, documentation, testing, and patches. Follow these steps to contribute:
+
+1. **Fork the Repository**
+
+2. **Set Up Git Hooks**
+
+    `git config core.hooksPath .githooks`: This command configures Git to use the `.githooks` directory for all Git hooks.
+
+3. **Create a Branch**
+
+4. **Commit Your Changes**
+
+5. **Submit a Pull Request**: Once you're satisfied with your changes, submit a pull request to the original SRI repository. Provide a clear and concise description of the changes you've made. If your pull request addresses an existing issue, reference the issue number in the description. To contribute to the protocol implementation, every PR must be opened against the `dev` branch. To better understand the SRI branch structure, please refer to [this guide](https://github.com/stratum-mining/stratum/blob/main/RELEASE.md#principal-branches).
+
+6. **Review and Iterate**
+
+7. **Merge and Close**: Once your pull request has been approved and all discussions have been resolved, a project maintainer will merge your changes into the `main` branch. Your contribution will then officially become part of the project. The pull request will be closed, marking the completion of your contribution.
+
+### Your First Code Contribution
+
+> To contribute, a basic understanding of Git and GitHub is needed. If you're not familiar with them, check out [this resource](https://docs.github.com/en/get-started/start-your-journey/git-and-github-learning-resources) to learn how to use them.
+
+Not sure where to start contributing to SRI? You can begin by looking through issues labeled `good first issue`:
+
+* [Good first issue](https://github.com/stratum-mining/benchmarking-tool/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) - These issues typically require only a few lines of code and a test or two.
```

Comment:
I would add also the `help-wanted` label.

And I'd also reference the project roadmap.

I'm just comparing this with the `contributing.md` we have on `stratum` repo: https://github.com/stratum-mining/stratum/blob/0f0ee1e32a7cdeb9af83718364e0896b36744824/CONTRIBUTING.md?plain=1#L109